cli/data/playbooks/setup/metadata.obsah.yaml

```
@@ -0,0 +1,3 @@
+---
+help: |
+  Setup up a host to be installed.
```

https://grammarist.com/spelling/set-up-vs-setup/

```suggestion
  Set up a host to be installed.
```